.tinyenv

```diff
@@ -1,2 +1,2 @@
-VERSION=0.0.6
```

Comment:
This version is bumped when there's something to deploy. In this case it won't be necessary.

Reply:
The reason is to avoid to run the custom deployment of the previous version 0.0.6

Reply:
Oh I see! Makes sense